learn/tooling-guide/ballerina-shell.md

```diff
@@ -2,11 +2,197 @@
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Ballerina Shell
 permalink: /learn/tooling-guide/ballerina-shell/
+description: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 active: ballerina-shell
-intro: The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger. Thus, you can run or debug your Ballerina programs easily via the VS Code Ballerina extension by launching its debugger. 
+intro: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 redirect_from:
   - /learn/tooling-guide/ballerina-shell
 ---
 
+It allows you to evaluate snippets of code without having to write complete programs. 
 
+This will particularly be your interest if you are looking to try out the language and its new features, and also for debugging purposes.
 
+## Setting up the Prerequisites
+
+The Ballerina Shell is a part of the Ballerina distribution. It was first shipped with Ballerina Swan Lake Alpha1. Therefore, if you are using Swan Lake Alpha1 or a later version, you are all set. If not, for information on installing the latest version of Ballerina, see [downloads](https://ballerina.io/downloads/).
+
+## Starting the Shell
+
+Execute the command below to start Ballerina Shell.
+
+```bash
+$ bal shell
+Welcome to Ballerina Shell REPL.
+Type /exit to exit and /help to list available commands.
+
+=$ 
+```
+
+## Stopping the Shell
+
+Execute the `/exit` command below in a new prompt to exit the Shell.
+
+```bash
+=$ /exit                                                                                                                                                                                                           
+| Bye!!!
+```
+
+## Evaluating Code Snippets
+
+The types of code snippets below are supported in the Shell.
```

Comment:
```suggestion
The following types of code snippets are supported in the Shell.
```

Reply:
"Following" should be used to refer to something that is mentioned immediately (e.g., after the column in the same line). The word "below" is more appropriate here as the code snippet is shown separately underneath the sentence.